src/request/plan.rs

```diff
@@ -574,13 +586,15 @@ where
                 "CleanupLocks::execute, meet locks:{}",
                 locks.len()
             );
-            result.meet_locks += locks.len();
 
+            let lock_size = locks.len();
             match lock_resolver
                 .cleanup_locks(self.store.clone().unwrap(), locks, self.pd_client.clone())
                 .await
             {
-                Ok(()) => {}
+                Ok(()) => {
+                    result.meet_locks += lock_size;
```

Comment:
I just recall that at that time I wrote these code, I expected `meet_locks` is the number of locks `cleanup_locks` "meet", and `resolved_locks` the number of "resolved" (there is a "TODO" in `CleanupLocksResult`).

I think it would be better to rename `meet_locks` to `resolved_locks` for recording the number of locks here.

Moreover, if CI is still flaky, we can just remove the check of `meet_locks`. The `count_locks` should be enough for correctness. It's not the issue of this PR. And we can polish them later.

Reply:
Fixed,  renamed `meet_locks` to `resolved_locks`.